doc/modules/model_evaluation.rst

```diff
@@ -2826,6 +2826,48 @@ Here are some usage examples of the :func:`d2_absolute_error_score` function::
 
 |details-end|
 
+|details-start|
+**D² log loss score**
+|details-split|
+
+The :func:`d2_log_loss_score` function implements the special case
+of D² with the log loss, see :ref:`log_loss`, i.e.:
+
+.. math::
+
+  \text{dev}(y, \hat{y}) = \text{log_loss}(y, \hat{y}).
+
+Here are some usage examples of the :func:`d2_log_loss_score` function::
```

Comment:
Please mention that the y_null for the log loss is the per-class frequency, here or in the introdction paragraph above where the y_null for other losses are described.